src/renderer/components/repositoryListView/repositoryList.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+	<div style="width: 100%">
+		<t-scrollbar v-if="getAllRepository.length > 0" height="300px">
+			<div style="padding: 1rem">
+				<div v-if="getAllRepository.length > 0">
+					<repository-item
+						v-for="repo in getAllRepository"
+						:key="repo.path"
+						:repo="repo"
```

Comment:
Can you split `repo` props into individual props?

* `repositoryName` - The name of the repository
* `index` - The index value from the array